app/commands/command/batch/publish.rb

```diff
@@ -0,0 +1,16 @@
+module Command
+  module Batch
+    class Publish
+      def initialize(collection, ids)
+        @collection = collection
+        @ids = ids
+      end
+
+      def call
+        @collection
+          .where(id: [*@ids])
+          .update_all(visibility_status: 'published')
```

Comment:
`update_all` won't call any callbacks, and when we merge public_activity PR we either need to add one activity like "George published x Litigations" and put ids in the comment. Or fire `@collection.where(id: [*ids]).update(visibility_status: 'published')`.

Reply:
Which will generate bunch of activites. Not sure what is better, I think just one batch activity seems fine.

Reply:
yeah, I think one batch activity would work well, and keep it saner. I'll update it.

Reply:
actually, let's leave it like this and then we address capturing these on the activities.